src/guidellm/backend/openai.py

```diff
@@ -128,12 +140,19 @@ def available_models(self) -> List[str]:
         :rtype: List[str]
         """
 
-        models: List[str] = [
-            model.id for model in self.openai_client.models.list().data
-        ]
-        logger.info(f"Available models: {models}")
-
-        return models
+        try:
+            models: List[str] = [
+                model.id for model in self.openai_client.models.list().data
+            ]
+            logger.info(f"Available models: {models}")
+        except openai.NotFoundError as error:
+            logger.error(error)
+            if settings.debug is True:
```

Comment:
So since we choose the llama.cpp as our primary openai serving engine it doesn't have anything for models.list().

Also if you serve the model with your llama.cpp it does not matter which model you request. And this is the actual reason why I added the settings.debug parameter. So we at least can run it locally.

If the concern is only about the gpt-4o we can rename it.